znai-reactjs/src/doc-elements/latex/InlinedLatex.demo.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 TWO SIGMA OPEN SOURCE, LLC
```

Comment:
2021 znai maintainers
there is a way to set this up in IDE so it auto generates these things